reaktive/src/jsMain/kotlin/com/badoo/reaktive/npmPublishTestSandbox.kt

```diff
@@ -0,0 +1,18 @@
+package com.badoo.reaktive
+
+import kotlin.js.Promise
+
+@JsExport
```

Comment:
Can we instead of this sample code being put in production folder, just js-export some real Reaktive things? E.g. the `Observable` interface and all its super types. To check the whole use case, we can create the following class in `sample-mpp-module` module:

```kotlin
@JsExport
class ExportedThing {
    val data: Observable<String> = observableOf("one", "two", "three")
}
```

Then we could also npm-publish the sample module and check how it can be used in a JS project. Would this work/help?

Reply:
I tried, but it's quite a rabbit hole once you start annotating the trail, so I decided to leave that to you guys, since you know your domain best.